### PR TITLE
DOC-9180 Add support for ARM platforms.

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

File: modules/install/pages/install-platforms.adoc [hunk 1]
```diff
@@ -105,6 +105,26 @@ Follow the best practices to run xref:best-practices-vm.adoc[Couchbase Server on
...
+
+.Supported ARM platforms
+|===
+| Operating system | Couchbase Version | Notes
+
+| Amazon  Linux 2 (64 bits)
+| 7.1
+| {empty}
+
+| Linux ARM
```
Comment (b33f) [inline modules/install/pages/install-platforms.adoc]: Won't it be confusing to say that "Linux ARM" is not supported when Amazon Linux on ARM is supported ? 
IMHO we should remove this note.

File: modules/install/pages/install-platforms.adoc [hunk 2]
```diff
@@ -105,6 +105,26 @@ Follow the best practices to run xref:best-practices-vm.adoc[Couchbase Server on
...
+| Not supported
+| {empty}
+
+| MacOS 11+
+| Not supported
```
Comment (b33f) [inline modules/install/pages/install-platforms.adoc]: Couchbase Server is supported on MacOS Apple Silicon (ARM) when using the MacOS Rosetta2 emulation layer.

**Review by b33f — CHANGES_REQUESTED** [review]

**Review by b33f — APPROVED** [review]

File: modules/install/pages/install-platforms.adoc [hunk 3]
```diff
@@ -105,6 +105,22 @@ Follow the best practices to run xref:best-practices-vm.adoc[Couchbase Server on
...
+[#supported-ARM-platforms]
+== ARM Platforms
+
+.Supported ARM platforms
+|===
+| Operating system | Couchbase Version | Notes
+
+| Amazon  Linux 2 (64 bits)
+| 7.1
+| {empty}
```
Comment (b33f) [inline modules/install/pages/install-platforms.adoc]: Requires a ARMv8 CPU such as the Amazon Graviton2

**Review by b33f — APPROVED** [review]

File: modules/release-notes/pages/relnotes.adoc [hunk 4]
```diff
@@ -381,6 +381,8 @@ This release adds support for the following platforms:
 
 * macOS Big Sur for development only
 
+* Amazon Linux (ARM )
```
Comment (simon-dew) [inline modules/release-notes/pages/relnotes.adoc]: Looks like a minor typo

```suggestion
* Amazon Linux (ARM)
```

**Review by simon-dew — COMMENTED** [review]
One minor typo I think? LGTM otherwise

**Review by simon-dew — APPROVED** [review]
LGTM 👍

**Comment by RayOffiah** [comment]
This is meant for the Neo release, so I'm closing the PR and creating another one for the 7.1 branch

**Comment by RayOffiah** [comment]
Closing and moving to Neo

**Pull request closed without merging**